src/pages/Validator/AutocompleteList.js

```diff
@@ -2,36 +2,78 @@ import React from "react";
 
 export default function AutocompleteList({
   entities = {},
+  showAttributes,
```

Comment:
Right now this is named like a bool or a function but its a string. Could we change it to something like `autocompleteTarget` or `entityName`?

Reply:
I like `entityName`, simple and string-y